todo/App.js

```diff
@@ -17,26 +20,24 @@ export default function App() {
     {text: 'play on the switch', complete: true, key: '3'},
   ]);
 
+  const [modalOpen, setModalOpen] = useState(false);
+  const [vall, setVall] = useState([]);
```

Comment:
give meaningful name to variable & functionn
Initial value should be an object I guess